pages/api/v1/expertBody/becomeExpert.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { NextApiHandler } from 'next';
+import storeApiKeyUsage from '~src/api-middlewares/storeApiKeyUsage';
+import withErrorHandling from '~src/api-middlewares/withErrorHandling';
+import { isValidNetwork } from '~src/api-utils';
+import authServiceInstance from '~src/auth/auth';
+import { MessageType } from '~src/auth/types';
+import getTokenFromReq from '~src/auth/utils/getTokenFromReq';
+import messages from '~src/auth/utils/messages';
+import { firestore_db } from '~src/services/firebaseInit';
+import { EExpertReqStatus } from '~src/types';
+import getEncodedAddress from '~src/util/getEncodedAddress';
+
+const handler: NextApiHandler<MessageType> = async (req, res) => {
+	storeApiKeyUsage(req);
+
+	try {
+		const network = String(req.headers['x-network']);
+		if (!network || !isValidNetwork(network)) return res.status(400).json({ message: messages.INVALID_NETWORK });
+
+		const { contribution, userAddress, reason } = req.body;
+
+		const encodedUserAddress = getEncodedAddress(userAddress, network);
```

Comment:
check if this address belongs to the user or not

Reply:
ok